src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -250,9 +271,9 @@ static void Main()
 ";
             var comp = CreateCompilationWithMscorlib(source, parseOptions: TestOptions.Regular.WithTuplesFeature());
             comp.VerifyDiagnostics(
-                // (8,18): error CS8208: The Deconstruct method for type 'C.Deconstruct(ref int, out int)' must have only out parameters.
+                // (8,9): error CS1620: Argument 1 must be passed with the 'ref' keyword
```

Comment:
The error is misleading. I think we should just say that Deconstruct is missing.